dandi/dandiapi.py

```diff
@@ -693,6 +693,10 @@ class RemoteValidationError(APIBase):
     message: str
 
 
+class RemoteAssetValidationError(RemoteValidationError):
+    path: str
```

Comment:
can we make pydantic assign path to be `None` (make it `Optional`?) if not present in current version? That is to not need to align 100% to version of server switch since there would be period of time when staging would be having it new way while main one old way.  And only whenever current version of dandi-archive is gone from use - would make it "mandatory".

Reply:
Done: 43d1d27